content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -1,18 +1,115 @@
+   although advanced users can specify requirements differently by modifying the
+   script.
+
+[script]: #submit-an-experiment
+
+### Setup Credentials
+
+To enable cloud experiments, first add credentials (or use credentials from a
+[data remote]). To add credentials, click on your user icon in the top right
+corner and go to your [Profile]. Navigate to [Cloud Credentials] and click
+`Add credentials`. Select either AWS, Azure, or GCP as the provider. Depending
+on the provider, you will be asked for more details. Finally, click
+`Save credentials`.
+
+![](https://static.iterative.ai/img/studio/s3_remote_settings_v2.png)
+
+The credentials must have the required permissions. See details and examples for
+each provider below:
+
+<details>
+
+#### AWS
+
+See
+https://github.com/iterative/terraform-provider-iterative/blob/a92499539f109821c021d1efb1fb01e51f1db47f/docs/guides/permissions/aws/main.tf
+
+</details>
+
+<details>
+
+#### Azure
 
-You can change your hyperparameters or select a different dataset and re-run
-your model training using Iterative Studio.
+See
+https://github.com/iterative/terraform-provider-iterative/blob/a92499539f109821c021d1efb1fb01e51f1db47f/docs/guides/permissions/az/main.tf
 
-## How Iterative Studio runs ML experiments
+</details>
+
+<details>
+
+#### GCP
+
+See
+https://github.com/iterative/terraform-provider-iterative/blob/a92499539f109821c021d1efb1fb01e51f1db47f/docs/guides/permissions/gcp/main.tf
+
+</details>
+
+[data remote]:
+  /doc/studio/user-guide/projects-and-experiments/configure-a-project#credentials
+[profile]: https://studio.iterative.ai/user/_/profile
+[cloud credentials]:
+  /doc/studio/user-guide/projects-and-experiments/account-management
+
+### Submit a new experiment
+
+Once you have added credentials, navigate to the project and select the Git
+commit from which you want to iterate. Click the button at the top shaped like a
+flask to open the form to run an experiment.
+
+At the top of the form, configure the cloud instance (including the region,
+size, GPU, HDD, timeout, and whether to use a spot instance). Under the
+`Parameters` tab, optionally modify any parameters of your [DVC experiment
+pipeline]. Under the `Script` tab, you can see the commands that will be run on
+the cloud instance and optionally modify them (this is not recommended unless
+you know what you are doing since it could cause the experiment to fail). At the
+bottom of the form, click `Run Experiment` to start the experiment.
+
+[dvc experiment pipeline]: /doc/start/experiments/experiment-pipelines
+
+### Monitor a running experiment
+
+Once you submit an experiment, the instance is created and the job script is
+run. A new row is created in the experiments table under the original Git
+commit. [Live updates to metrics and plots] generated by [DVCLive] will show up
+in this row, and you can click on the experiment name to view the status and
+output log of the running experiment task.
+
```

Comment:
TODO: Add a screenshot or gif showing experiment metrics updates and logs.

Reply:
I'll add this. I have gif from the past, and if there are changes, I'll create a new one.

Reply:
Added this. I'm using a different project from what you're using though.

Reply:
Feel free to replace my image if you want to keep it consistent.